cookbooks/rubygems-app/recipes/logrotate.rb

```diff
@@ -11,6 +11,7 @@
   options %w(missingok nocreate compress delaycompress dateext notifempty sharedscripts)
   postrotate [
     '    [ -f /etc/service/unicorn/supervise/pid ] && kill -USR1 `cat /etc/service/unicorn/supervise/pid`',
-    '    [ -f /etc/service/delayed_job/supervise/pid ] && kill -USR1 `cat /etc/service/delayed_job/supervise/pid`'
+    '    [ -f /etc/service/delayed_job/supervise/pid ] && kill -USR1 `cat /etc/service/delayed_job/supervise/pid`',
+    '    [ -f /etc/service/shoryuken/supervise/pid ] && kill -USR1 `cat /etc/service/shoryuken/supervise/pid`'
```

Comment:
Sounds like a fun thing to debug. :laughing:

I updated the PR so delayed_job and shoryuken use similar restart commands.